app/components/ClockMinimal.js

```diff
@@ -0,0 +1,22 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('ClockMinimal', {
+  className: 'tilt-icon-clock-minimal',
+
+  render() {
+    return (
+      <svg className={this.getClassNames()} viewBox="0 0 23 23" >
+        <title>ClockMinimal</title>
+        <g className="tilt-icon-main" transform="translate(-511.000000, -1908.000000)">
```

Comment:
you can add up all the translates into a single declaration and get rid of some unnecessary `<g/>`s
```
-511 + 484 + 27 = 0
-1908 + 1901 + 7 = 0
```

in this case we don't even need to declare the transforms.